webapp/src/views/customers_list.php

```diff
@@ -5,8 +5,26 @@
 <div class=" mx-auto" style="margin-top: 30px; width: 90%">
     <h3 class="title text-center mb-md-4 mb-sm-3 mb-3 mb-2">
         <?php echo $title ?></h3>
-
-
+        <div class="search-header">
+        <div class="radio">
+            <div class="form-check form-check-inline">
+                <label class="form-check-inline">
+                <input type="radio" class="form-check-input" checked name="searchType" value="1">
+                <h4><?php echo localize('Personal-Phone');?></h4></label>
+                <input id="phoneNumber" type="text" class="form-control search-bar" 
+                name="phoneNumber" placeholder='<?php echo localize('Personal-Phone'); ?>' />
+            </div>
+            <div class="form-check form-check-inline">
+                <label class="form-check-inline">
+                <input type="radio" class="form-check-input" name="searchType" value="2">
+                <h4><?php echo localize('Inscription-Lastname'); ?></h4></label>
+                <input id="customerName" type="text" class="form-control search-bar" 
+                name="customerName" placeholder='<?php echo localize('Inscription-Lastname'); ?>' />
+            </div>
+            <input type="button" value="<?php echo localize("searchClient"); ?>" class="form-check-inline btn btn-success" onclick="SearchCustomer();">
```

Comment:
Dans les fichiers de localize, on commence toujours par une lettre majuscule. U know what to do now ;)